hbase-client/src/main/java/org/apache/hadoop/hbase/client/MutableRegionInfo.java

```diff
@@ -254,8 +254,10 @@ public MutableRegionInfo setSplit(boolean split) {
 
   /**
    * @return True if this region is offline.
+   * @deprecated since 3.0.0 and will be removed in 4.0.0
```

Comment:
Please add comments to explain the reason.

Reply:
What @Apache9 said..... say, is only used when we split a region so split == offline.